app/controllers/miq_policy_log_controller.rb

```diff
@@ -0,0 +1,95 @@
+class MiqPolicyLogController < ApplicationController
+  before_action :check_privileges
+  before_action :get_session_data
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  include Mixins::GenericSessionMixin
+  include Mixins::BreadcrumbsMixin
+
+  def index
+    flash_to_session
+    assert_privileges('policy_log')
+    @breadcrumbs = []
+    @log = $policy_log.contents(nil, 1000)
+    add_flash(_("Logs for this %{product} Server are not available for viewing") % {:product => Vmdb::Appliance.PRODUCT_NAME}, :warning) if @log.blank?
+    @lastaction = "policy_logs"
+    @layout = "miq_policy_logs"
+    @msg_title = "Policy"
+    @download_action = "fetch_log"
+    @server_options ||= {}
+    @server_options[:server_id] ||= MiqServer.my_server.id
+    @server = MiqServer.my_server
```

Comment:
This is a little bit too many instance variables, do we really need all of them? I'm okay with a cleanup in a future PR, just please create an issue to track this (and maybe the rest of the controllers) if you decide so. Or you can do it here, up to you...

Reply:
@skateman cleaned up some of the instance variables, please take a look, if further cleanup is needed, i can open an issue for that.